src/fragments/lib/auth/flutter/getting_started/30_initAuth.mdx

```diff
@@ -9,35 +9,26 @@ import 'package:flutter/material.dart';
 
 import 'amplifyconfiguration.dart';
 
-void main() {
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await _configureAmplify();
```

Comment:
I would leave this out until we look into reducing configuration time.

Reply:
Yep, good thinking. Will update.